src/pycram/ros/tf_broadcaster.py

```diff
@@ -78,8 +79,8 @@ def _publish_pose(self, child_frame_id: str, pose: Pose, static=False) -> None:
         frame_id = pose.frame
         if frame_id != child_frame_id:
             tf_stamped = pose.to_transform(child_frame_id)
-            tf_stamped.frame = self.projection_namespace + "/" + tf_stamped.frame
-            tf_stamped.child_frame_id = self.projection_namespace + "/" + tf_stamped.child_frame_id
+            tf_stamped.frame = self.projection_namespace.name + "/" + tf_stamped.frame
```

Comment:
isnt os.path.join nicer here?

Reply:
Not really the scope of this PR, since this is only fixing a bug, but tried it nevertheless. `os.path.join` behaves differently depending on the platform. No problem if we are always on Linux with this. I could add `"/".join(...)` though.

Reply:
Bug fixes are also required to be as clean as possible.
The question was more If for another os than Linux ghe rostopic still has "/" as seperator or If eg on the seperator will be "\"

Reply:
*Backslash